troposphere/settings/local.py.j2

```diff
@@ -102,6 +112,10 @@ BADGE_SYSTEM_NAME = "{{ BADGE_SYSTEM_NAME }}"
 BADGE_SECRET = "{{ BADGE_SECRET }}"
 {% endif %}
 
+# resolved JS asset + hash for template rendering
+INSTALLED_APPS += (
+    'webpack_loader',
+)
```

Comment:
If this is going to be a requirement of troposphere, default.py may be a better location..